src/lib/operators/alias_operator.cpp

```diff
@@ -22,10 +22,11 @@ const std::string& AliasOperator::name() const {
 }
 
 std::string AliasOperator::description(DescriptionMode description_mode) const {
-  const auto* const separator = description_mode == DescriptionMode::SingleLine ? " " : "\n";
+  const char separator = (description_mode == DescriptionMode::SingleLine ? ' ' : '\n');
   std::stringstream stream;
 
-  stream << AbstractOperator::description(description_mode) << separator << "[";
+  stream << AbstractOperator::description(description_mode) << separator;
+  stream << "[";
```

Comment:
That added line looks unnecessary. Is it added by the linter?

Reply:
No, added for readability reasons.
Maybe this fits better:

```cpp
stream << "[" << boost::algorithm::join(_aliases, ", ") << "]";
```

Reply:
Hmm, looking at it again I can see your point.

Do whatever pleases your eye. ;)